dialog.js

```diff
@@ -229,6 +229,9 @@ async function setDisplayFields(result, urlFieldId, contentTypeFieldId,
       document.getElementById(imgFieldId).innerHTML = "✅";
   } else {
       document.getElementById(imgFieldId).innerHTML = "❌";
+      if (urlFieldId === "url") {
```

Comment:
Alternatively, default to `display:none` and change to `block` if we have something to show? Whichever looks less confusing.

Reply:
I think it's better to leave on by default (which should be the common case for SXGs)  and only remove it when there's an error.